esphome/components/web_server_base/__init__.py

```diff
@@ -25,5 +25,5 @@ async def to_code(config):
 
     if CORE.is_esp32:
         cg.add_library("FS", None)
-    # https://github.com/OttoWinter/ESPAsyncWebServer/blob/master/library.json
-    cg.add_library("ESPAsyncWebServer-esphome", "1.2.7")
+    # https://github.com/esphome/ESPAsyncWebServer/blob/master/library.json
+    cg.add_library("ESPAsyncWebServer-esphome", "1.3.0")
```

Comment:
```suggestion
    cg.add_library("esphome/ESPAsyncWebServer-esphome", "1.3.0")
```

To restrict it to our package